__tests__/commands/authError.test.ts

```diff
@@ -0,0 +1,77 @@
+import { describe, expect, test } from '@jest/globals';
+import { ERRORS } from '../../src/constant';
+import {
+  appIDValidator,
+  authValidator,
+  domainValidator,
+  passwordValidator,
+  proxyValidator,
+  usernameValidator
+} from '../../src/commands/Auth/validator';
+
+describe('auth command: validator', () => {
+  test(`The name app missing -> ${ERRORS.APP_NAME_MISSING}`, () => {
+    const input = '';
+    expect(authValidator(input)).toBe(ERRORS.APP_NAME_MISSING);
+  });
```

Comment:
Thanks for modifying following the suggestion.
One more thing...
I think we should use `` quotation in case using variables. So in this case you can improve it like this.
```
// Before
`Should have domainValidator as 'Domain has to start with https'`

// After
'Should have domainValidator as "Domain has to start with https"'
```

Reply:
And about the description, I cannot know the case from it.
Please add the info of "in case not setting app name" for example.
Other parts are the same.